docs/release-notes/2.33.0.md

```diff
@@ -13,28 +13,28 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 ## Release Highlights
 
 
-### Data loss prevention for PMM server outages
+### Offline metric collection during PMM server outages or loss of PMM client-server network connectivity
```

Comment:
Do we really need this part on title section ? "or loss of PMM client-server network connectivity"

Reply:
I think Offline metric collection is better. It give clarity.